Alpaca-Finance-Agents/Liquidation-Alert/src/agent.ts

```diff
@@ -0,0 +1,108 @@
+import { 
+  Finding, 
+  HandleTransaction, 
+  TransactionEvent, 
+  FindingSeverity, 
+  FindingType,
+} from 'forta-agent';
+
+const VAULT_ADDRESSES: string[] = [
+  "0x7C9e73d4C71dae564d41F78d56439bB4ba87592f".toLowerCase(), // BUSD
+  "0xbfF4a34A4644a113E8200D7F1D79b3555f723AfE".toLowerCase(), // ETH
+  "0x08FC9Ba2cAc74742177e0afC3dC8Aed6961c24e7".toLowerCase(), // BTCB
+  "0x08FC9Ba2cAc74742177e0afC3dC8Aed6961c24e7".toLowerCase(), // USDT
+  "0xf1bE8ecC990cBcb90e166b71E368299f0116d421".toLowerCase(), // ALPACA
+  "0x3282d2a151ca00BfE7ed17Aa16E42880248CD3Cd".toLowerCase()  // TUSD
+]
+
+const killEventAbi: string = "event Kill(uint256 indexed id, address indexed killer, address owner, uint256 posVal, uint256 debt, uint256 prize, uint256 left)";
+
+const createFindings = (
+  posId: number,
+  killer: string,
+  posOwner: string,
+  posVal: number,
+  debt: number,
+  prize: number,
+  left: number,
+  vaultAddress: string
+): Finding[] => {
```

Comment:
I think you shouldn't delegate the responsibility for deciding which alerts to emit to this function.